curator-framework/src/test/resources/schema1.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "name": "test",
```

Comment:
The tests would read a bit easier if the name hinted to the contents of the schema, like `ephemeralNotSequential`?